Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM sonatype/nexus3:3.1.0
+MAINTAINER Dwolla Dev <dev+docker-nexus3-crowd@dwolla.com>
+LABEL org.label-schema.vcs-url="https://github.com/Dwolla/docker-nexus3-crowd"
+
+USER root
+
+# install git and maven
+RUN yum install -y git && \
+    curl -o /etc/yum.repos.d/epel-apache-maven.repo https://repos.fedorapeople.org/repos/dchen/apache-maven/epel-apache-maven.repo && \
+    yum install -y apache-maven
+
+# install nexus3-crowd-plugin
+RUN git clone https://github.com/pingunaut/nexus3-crowd-plugin.git && \
+    cd nexus3-crowd-plugin && \
+    git checkout tags/nexus3-crowd-plugin-3.2.0 && \
+    mvn install && \
+    cp -ra ~/.m2/repository/com/pingunaut /opt/sonatype/nexus/system/com && \
+    echo -e 'mvn\:com.pingunaut.nexus/nexus3-crowd-plugin/3.2.0 = 200' >> /opt/sonatype/nexus/etc/karaf/startup.properties
+
+# remove git and maven
+RUN yum remove -y git && \
```

Comment:
I don't think this will actually shrink the image because these commands are in a different `RUN` statement than the ones that add the files. If we consolidate the three `RUN` statements into a single statement, it should work better.